manifests/board-image/openkylin-bpi-f3/0.2.0-SP1.toml

```diff
@@ -0,0 +1,30 @@
+format = "v1"
+[[distfiles]]
+name = "openKylin-Embedded-V2.0-SP1-spacemit-k1-riscv64...>"
```

Comment:
The filename looks not okay.

Reply:
I'll map the openKylin's version in following rules: if is semver, convert -SP* to +sp* if has, otherwise seen -SP* as a patch version

Reply:
> I'll map the openKylin's version in following rules: if is semver, convert -SP* to +sp* if has, otherwise seen -SP* as a patch version

This is also confusing, because `0.2.0 == 0.2.0+sp1` (build tags do not take part in comparison).